### ISSUE TITLE
Documentation about snippets are wrong

### ISSUE THREAD

**Issue by drewhk** [lightbend/paradox#72]
(version 0.2.7)
The docs here says (http://developer.lightbend.com/docs/paradox/latest/features/snippet-inclusion.html):

```
sbt
:   @@snip [build.sbt](/../../../build.sbt) { #setup_example }
Maven
:   @@snip [pom.xml](../../../pom.xml) { #setup_example }
```

But in reality you need a newline:

```
sbt
:   @@snip [build.sbt](/../../../build.sbt) { #setup_example }

Maven
:   @@snip [pom.xml](../../../pom.xml) { #setup_example }
```

**Comment by 2m** [comment]
Fixed in #75

**Comment by 2m** [comment]
This should be closed and assigned to the next milestone.

**Comment by dwijnand** [comment]
Shipped in v0.2.8
